src/spaceObjects/cpuShip.cpp

```diff
@@ -286,17 +286,17 @@ string getAIOrderString(EAIOrder order)
 {
     switch(order)
     {
-    case AI_Idle: return "Idle";
-    case AI_Roaming: return "Roaming";
-    case AI_Retreat: return "Retreat";
-    case AI_StandGround: return "Stand Ground";
-    case AI_DefendLocation: return "Defend Location";
-    case AI_DefendTarget: return "Defend Target";
-    case AI_FlyFormation: return "Fly in formation";
-    case AI_FlyTowards: return "Fly towards";
-    case AI_FlyTowardsBlind: return "Fly towards (ignore all)";
-    case AI_Attack: return "Attack";
-    case AI_Dock: return "Dock";
+    case AI_Idle: return tr("orderscpu", "Idle");
```

Comment:
Sadly, we cannot do this like this. As the `getAIOrderString` function is also used for the lua scripting API, so this will break things if a different language is selected.

SystemName had the same issue, see how we solved it here: https://github.com/daid/EmptyEpsilon/blob/master/src/shipTemplate.cpp#L374